static/pricing-page/stylesheets/styles.css

```diff
@@ -11,7 +11,7 @@
 .PricePage {
   position: relative;
   width: 100%;
-  height: 2700px;
+  height: 2750px;
```

Comment:
As per https://github.com/opencollective/opencollective/issues/2460#issuecomment-537947964

> So the main problem here is that we have `height: 2700px;` on the pricing page. That will never work on mobile.
> 
> Whatever the fix, we must get rid of this fixed height.

We need to stop doing that, we'll have other problems in the future otherwise.

Reply:
The problem with this, removing it collapses everything. The layout of that page too dependent on `absolute` position.

But I think it the page works pretty well on mobile currently.

Reply:
> The problem with this, removing it collapses everything. The layout of that page too dependent on absolute position.

That seems very hacky to me, the layout of this page is pretty standard.

> But I think it the page works pretty well on mobile currently.

The FAQ layout is still incorrect on mobile:

![image](https://user-images.githubusercontent.com/1556356/66271987-5d3a8280-e864-11e9-91da-7f88372a08e9.png)

Reply:
I'll see to this and remove the height.